src/browser/commands.js

```diff
@@ -0,0 +1,45 @@
+import { sendKeys as cmdSendKeys, sendMouse } from '@web/test-runner-commands';
+
+function getElementPosition(elem) {
+	const { x, y, width, height } = elem.getBoundingClientRect();
+	return [
+		Math.floor(x + window.scrollX + width / 2),
+		Math.floor(y + window.scrollY + height / 2),
+	];
```

Comment:
_Super_ minor, but if this was an object with `x` and `y` instead, would make the lines below a tad prettier
```
return clickAt(position.x, position.y);
```